src/types/dispute-status-types.js

```diff
@@ -31,16 +31,16 @@ const stringMapping = {
   [Status.Closed]: 'Closed',
   [Phase.All]: 'All',
   [Phase.Evidence]: 'Adding arguments',
-  [Phase.JuryDrafting]: 'Jury drafting',
+  [Phase.JuryDrafting]: 'Guardians summon',
```

Comment:
```suggestion
  [Phase.JuryDrafting]: 'Summon guardians',
```

Perhaps this sounds better haha

Reply:
hahah ahh thanxs for your review, i was going to ask paty to review the summon wording i have lot of dubts, pretty weird word for me haha